client/components/__mocks__/app.js

```diff
@@ -0,0 +1,3 @@
+// const listing = {
```

Comment:
Hm, in a guide I was following, it stated that "module.exports = 'test-file-stub';" was needed here, but is this working fine as is?

Reply:
My mock isn't actually up and running. I may make use of your tip there!